jobs/resource_job.go

```diff
@@ -118,6 +118,46 @@ type RunJobTask struct {
 	JobParameters map[string]string `json:"job_parameters,omitempty"`
 }
 
+// TODO: migrate to go sdk
```

Comment:
nit, add a note stating that TF does not support recursive nesting and that we need to limit the depth. Maybe add a link to an example:

https://github.com/hashicorp/terraform-provider-aws/blob/b4a9f93a2b7323202c8904e86cff03d3f2cb006b/internal/service/wafv2/rule_group.go#L110